java/ql/src/Security/CWE/CWE-925/ImproperIntentVerification.qhelp

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+
+<qhelp>
+
+<overview>
+<p>
+When an android application uses a <code>BroadcastReciever</code> to receive Intents, 
+it is also able to receive explicit Intents that are sent drctly to it, egardless of its filter.
+
+Certain intent actions are only able to be sent by the operating system, not third-party applications. 
+However, a <code>BroadcastReceiver</code> that is registered to recieve system intents is still able to recieve 
+other intents from a third-party application, so it should check that the intent received has the expected action.
+Otherwise, a third-party application could impersonate the system this way and cause unintended behaviour, such as a denial of service.
```

Comment:
I think we use American English in qhelp and QLDoc.

```suggestion
Otherwise, a third-party application could impersonate the system this way and cause unintended behavior, such as a denial of service.
```